components/Starknet/modules/architecture-and-concepts/pages/network-architecture/messaging-mechanism.adoc

```diff
@@ -226,9 +227,22 @@ Where:
 
 [NOTE]
 ====
-In an `l1_handler` transaction, the first element of the calldata is always the Ethereum address of the sender.
+In a L1 handler transaction, the first element of the calldata is always the Ethereum address of the sender.
```

Comment:
I would stick with "an L1 handler". When I read this, I say "el one handler", which suggest "an", not "a".